src/Compilers/Core/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -113,46 +49,11 @@ End Sub
 End Module")
         };
 
-        private static readonly string[] s_allCompilerFiles =
-        {
-            s_csharpCompilerExecutableSrc,
-            s_basicCompilerExecutableSrc,
-            s_compilerServerExecutableSrc,
-            s_microsoftCodeAnalysisDllSrc,
-            s_systemCollectionsImmutableDllSrc,
-            s_buildTaskDllSrc,
-            ResolveAssemblyPath("System.Reflection.Metadata.dll"),
-            ResolveAssemblyPath("Microsoft.CodeAnalysis.CSharp.dll"),
-            ResolveAssemblyPath("Microsoft.CodeAnalysis.VisualBasic.dll"),
-            Path.Combine(s_clientExecutableBasePath, CompilerServerExeName + ".config"),
-            Path.Combine(s_clientExecutableBasePath, "csc.rsp"),
-            Path.Combine(s_clientExecutableBasePath, "vbc.rsp")
-        };
-
-        private readonly TempDirectory _tempDirectory;
-        private readonly string _compilerDirectory;
-
-        private readonly string _csharpCompilerClientExecutable;
-        private readonly string _basicCompilerClientExecutable;
-        private readonly string _compilerServerExecutable;
-        private readonly string _buildTaskDll;
+        ExecutingTestUtils.TempExecutionEnvironment _executionEnvironment = null;
 
         public CompilerServerUnitTests()
         {
-            _tempDirectory = Temp.CreateDirectory();
-
-            // Copy the compiler files to a temporary directory
-            _compilerDirectory = Temp.CreateDirectory().Path;
-            foreach (var path in s_allCompilerFiles)
-            {
-                var filename = Path.GetFileName(path);
-                File.Copy(path, Path.Combine(_compilerDirectory, filename));
-            }
-
-            _csharpCompilerClientExecutable = Path.Combine(_compilerDirectory, CSharpClientExeName);
-            _basicCompilerClientExecutable = Path.Combine(_compilerDirectory, BasicClientExeName);
-            _compilerServerExecutable = Path.Combine(_compilerDirectory, CompilerServerExeName);
-            _buildTaskDll = Path.Combine(_compilerDirectory, BuildTaskDllName);
+            executionEnvironment = new ExecutingTestUtils.TempExecutionEnvironment();
```

Comment:
Yep... it looks like I caught that at exactly the same time you did.  Really should have tested before pushing.

"but it's such a simple change" he said, not seeing the red squiggles.